base/applications/mspaint/dialogs.cpp

```diff
@@ -80,7 +80,10 @@ LRESULT CAttributesDialog::OnInitDialog(UINT nMsg, WPARAM wParam, LPARAM lParam,
     newHeight = imageModel.GetHeight();
 
     CheckDlgButton(IDD_ATTRIBUTESRB3, BST_CHECKED);
-    CheckDlgButton(IDD_ATTRIBUTESRB5, BST_CHECKED);
+    if (!paletteWindow.bk)
+        CheckDlgButton(IDD_ATTRIBUTESRB5, BST_CHECKED);
+    else
+        CheckDlgButton(IDD_ATTRIBUTESRB4, BST_CHECKED);
```

Comment:
same suggestion as in OnDefault() by the way.

Reply:
Ok, i have removed duplicated.